arangod/RocksDBEngine/RocksDBGeoIndex.h

```diff
@@ -136,7 +136,7 @@ class RocksDBGeoIndex final : public RocksDBIndex {
 
   bool matchesDefinition(VPackSlice const& info) const override;
 
-  int unload() override;
+  void unload() override {}
```

Comment:
Looks like it. However, it is already part of devel (my branch was slightly behind. If you want to remove it, please create a new bug-fix.